exercises/complex-numbers/complex_numbers_test.rb

```diff
@@ -0,0 +1,207 @@
+require 'minitest/autorun'
+require_relative 'complex_numbers'
+
+# Common test data version: 1.0.0 '117d062'
+class ComplexNumbersTest < Minitest::Test
+  def test_imaginary_unit
+    # #skip
+    expected = ComplexNumber.new([-1, 0])
+    assert_equal expected, ComplexNumber.new([0, 1]) * ComplexNumber.new([0, 1])
+  end
+
+  def test_add_purely_real_numbers
+    #skip
+    expected = ComplexNumber.new([3, 0])
```

Comment:
The array as an argument to ComplexNumber seems redundant, could we just do:
`ComplexNumber.new(3,0)`

Reply:
This is probably a good idea.